storm-client/src/jvm/org/apache/storm/StormSubmitter.java

```diff
@@ -435,7 +436,7 @@ public void onCompleted(String srcFile, String targetFile, long totalBytes) {
         });
     }
 
-    private static boolean topologyNameExists(String name, NimbusClient client) {
+    private static boolean isTopologyNameAllowed(String name, NimbusClient client) {
```

Comment:
Changing the name here because it calls `client.getClient().isTopologyNameAllowed` and it doesn't only check the existence of the topology name.